components/panel/style.scss

```diff
@@ -0,0 +1,60 @@
+.components-panel {
+	background: $white;
+	border: 1px solid $light-gray-500;
+
+	> .components-panel__header,
```

Comment:
I don't think we should optimize for reusing the panel pieces, only the panels themselves at this stage. If something is reusuable outside a panel, then it should be a separate component and get its own class for clarity.